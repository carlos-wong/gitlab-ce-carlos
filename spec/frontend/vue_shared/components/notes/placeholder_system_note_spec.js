import { shallowMount } from '@vue/test-utils';
import PlaceholderSystemNote from '~/vue_shared/components/notes/placeholder_system_note.vue';

describe('Placeholder system note component', () => {
  let wrapper;

  const createComponent = () => {
    wrapper = shallowMount(PlaceholderSystemNote, {
      propsData: {
        note: { body: 'This is a placeholder' },
      },
    });
  };

  afterEach(() => {
    wrapper.destroy();
    wrapper = null;
  });

  it('matches snapshot', () => {
    createComponent();

    expect(wrapper.element).toMatchSnapshot();
  });
});
