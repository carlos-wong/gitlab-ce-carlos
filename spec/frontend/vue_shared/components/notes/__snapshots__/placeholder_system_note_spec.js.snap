// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Placeholder system note component matches snapshot 1`] = `
<timeline-entry-item-stub
  class="note system-note being-posted fade-in-half"
>
  <div
    class="timeline-content"
  >
    <em>
      This is a placeholder
    </em>
  </div>
</timeline-entry-item-stub>
`;
